Resources/doc/config.rst

```diff
@@ -405,9 +405,10 @@ Otherwise you will get a *auth failed* exception.
             connections:
                 default:
                     server: "mongodb://localhost:27017"
-                    username: someuser
-                    password: somepass
-                    authSource: db_you_have_access_to
+                    options:
+                        username: someuser
+                        password: somepass
+                        db: db_you_have_access_to
```

Comment:
Even in the fixtures of 3.2.1 (at least, I haven't checked any other versions) I can't seem to find `authSource`

```
connections:
        conn1:
            server:               mongodb://localhost
            options:
                connect:          true
                connectTimeoutMS: 500
                db:               database_val
                journal:          true
                password:         password_val
                readPreference:   secondaryPreferred
                readPreferenceTags:
                  - { dc: east, use: reporting }
                  - { dc: west }
                  - {  }
                replicaSet:       foo
                slaveOkay:        true
                socketTimeoutMS:  1000
                ssl:              true
                authMechanism:    X509
                username:         username_val
                w:                majority
                wTimeoutMS:       1000
        conn2:
            server: mongodb://otherhost
```